code/modules/mob/living/silicon/robot/robot_model.dm

```diff
@@ -508,7 +508,8 @@
 	basic_modules = list(
 		/obj/item/assembly/flash/cyborg,
 		/obj/item/reagent_containers/glass/beaker/large, //I know a shaker is more appropiate but this is for ease of identification
-		/obj/item/reagent_containers/food/condiment/enzyme,
+		//Skyrat Edit Start: Borg Buff
```

Comment:
this isn't a buff, this is a nerf?

Reply:
It was a PR Jake made, it's included in one of the shakers underneath.

Reply:
ah